src/test/ui/consts/const_let_eq_float.rs

```diff
@@ -0,0 +1,279 @@
+// compile-pass
```

Comment:
Ooops... I think I made a mistake here; it should be `// run-pass` cause there's an `fn main() { ... }` at the end.